### ISSUE TITLE
Altair fails to load plugin

### ISSUE THREAD

**Issue by neominik** [XKojiMedia/altair-graphql-plugin-tracing#1]
Thanks for creating this plugin. Having tracing in Altair also would be amazing!

Unfortunately, when I add the plugin from the plugin manager (shows latest version 1.0.1) of the plugin in Altair (verison 4.0.11 installed from homebrew), the following error appears in the console:
```
Error fetching plugin assets Error: Could not create the plugin instance for plugin: altair-graphql-plugin-tracing. Check that plugin_class is set correctly in manifest.
    at main-es2018.js:1
    at Generator.next (<anonymous>)
    at i (main-es2018.js:1)
    at a (main-es2018.js:1)
    at main-es2018.js:1
    at new e (polyfills-es2018.js:1)
    at main-es2018.js:1
    at e.add (main-es2018.js:1)
    at main-es2018.js:1
    at Generator.next (<anonymous>)
```
I'm not exactly sure what is wrong here. The `plugin_class` seems to be set correctly and the sources for the plugin are fetched and visible in the dev tools. Any idea how to debug this?

**Comment by imolorhe** [comment]
should be fixed in 1.0.3

**Comment by neominik** [comment]
Can you publish a release, so it is easy to install in altair?
